src/tabs/tab-header-bar.tsx

```diff
@@ -19,6 +20,35 @@ import { useVisualRefresh } from '../internal/hooks/use-visual-mode';
 import { useInternalI18n } from '../i18n/context';
 import { useContainerQuery } from '@cloudscape-design/component-toolkit';
 
+function setTabIndexForTabContainer(
```

Comment:
I'm not sure if it's really necessary to move this into its own function if it's only called in one place.